code/modules/clothing/spacesuits/void/misc.dm

```diff
@@ -64,4 +64,82 @@
 	icon_state = "skrell_suit_white"
 
 /obj/item/clothing/suit/space/void/skrell/black
-	icon_state = "skrell_suit_black"
\ No newline at end of file
+	icon_state = "skrell_suit_black"
+
+/obj/item/clothing/head/helmet/space/void/dohruk/
+	name = "Unathi Helmet"
```

Comment:
damn it's been a while since I wrote all this. It being "unathi helmet" is intentional. There's sliiightly more to the helmet but it's intentional that the origins of the poachers aren't blatantly obvious to everyone. ima get the rest of this though